### PR TITLE
UserDefaults "Data" broken

### PR DESCRIPTION
**Pull request by borut-t**

### CODE REVIEW

File: Tests/Tests/Core/Utilities/UserDefaults/UserDefaultTests.swift [hunk 1]
```diff
@@ -36,17 +35,37 @@ class UserDefaultTests: XCTestCase {
...
+    
+    Defaults.profileData = givenValue
+    
+    XCTAssertNotNil(Defaults.profileData)
+  }
+  
+  func testSaveDataNullValue() {
+    let givenValue: Data? = nil
+    
+    Defaults.profileData = givenValue
```
Comment (borut-t) [inline Tests/Tests/Core/Utilities/UserDefaults/UserDefaultTests.swift:50]: This crashes. @EgzonArifi can you please take a look what's going on?

**Review by borut-t — COMMENTED** [review]
Thank you 🙌

File: Tests/Tests/Core/Utilities/UserDefaults/UserDefaultTests.swift (hunk 1 above)
Comment (dejanskledar) [reply Tests/Tests/Core/Utilities/UserDefaults/UserDefaultTests.swift:50]: Should be resolved.

**Review by dejanskledar — APPROVED** [review]

**Comment by borut-t** [comment]
Confirming it works 🙌

**Pull request merged**